configurations/test_pipeline_configuration.py

```diff
@@ -1,9 +1,18 @@
 from core_data_modules.cleaners import swahili
 
+from dateutil.parser import isoparse
+
+
 from src.pipeline_configuration_spec import *
 
 PIPELINE_CONFIGURATION = PipelineConfiguration(
     pipeline_name="engagement-db-test",
+    project_start_date = isoparse("2021-03-01T10:30:00+03:00"),
+    project_end_date = isoparse("2100-01-01T00:00:00+03:00"),
+    filter_test_messages = False,
+    test_contact_uuids = [
+        "f9158369-b985-4271-a9d6-5dd25e19e43c"
+    ],
```

Comment:
Do we want to apply these filters everywhere or just in analysis?

Reply:
At the moment yes - will keep this here and add them to the analysis configuration if no other module uses them.